### PR TITLE
NIOSH.ARMPS version 6.2.02

### PR DESCRIPTION
**Pull request by TechnicaVivunt**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/79612)

### CODE REVIEW

**Comment by TechnicaVivunt** [comment]
Possibly being blocked by the CDC: the URL for the download page is https://www.cdc.gov/niosh/mining/works/coversheet1813.html

File: manifests/n/NIOSH/ARMPS/6.2.02/NIOSH.ARMPS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NIOSH.ARMPS
+PackageVersion: 6.2.02
+PackageLocale: en-US
+Publisher: National Institute for Occupational Safety and Health
+PackageName: Analysis of Retreat Mining Pillar Stability
```
Comment (ItzLevvie) [inline manifests/n/NIOSH/ARMPS/6.2.02/NIOSH.ARMPS.locale.en-US.yaml:8]:
```suggestion
PackageName: ARMPS
```
This must match Control Panel.

File: manifests/n/NIOSH/ARMPS/6.2.02/NIOSH.ARMPS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: NIOSH.ARMPS
+PackageVersion: 6.2.02
+PackageLocale: en-US
+Publisher: National Institute for Occupational Safety and Health
```
Comment (ItzLevvie) [inline manifests/n/NIOSH/ARMPS/6.2.02/NIOSH.ARMPS.locale.en-US.yaml:7]:
```suggestion
Publisher: NIOSH
```
This must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**